### PR TITLE
Dockerfile-edge: added 7.1 trial (edge) container

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,45 +10,49 @@ Ubuntu used by default, Alpine builds also available tagged as `-alpine`
 - Nginx
```
Comment (bryanlatten) [inline README.md:13]: alphabetized a few that weren't

File: Dockerfile-edge [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        php7.1-mbstring \
+        php7.1-mcrypt \
+        php7.1-zip \
+        php-apcu \
+        php-gearman \
+        php-igbinary \
+        php-memcache \
+        php-memcached \
+        php-xdebug \
+        newrelic-php5 \
```
Comment (jimdelois) [inline Dockerfile-edge:57]: `php5` ?

Comment (bryanlatten) [inline Dockerfile-edge:57]: https://discuss.newrelic.com/t/php-agent-and-php-7-0/27687/79

**Comment by jimdelois** [comment]
👍

**Comment by mannytoledo** [comment]
👍 [root@bepro11 ~]# gearadmin -p 4730 -h localhost --status
expireUserGrace 0       0       10
recurringIntegrity      0       0       1
checkUserProvision      0       0       10
deletedIntegrity        0       0       10

**Comment by mannytoledo** [comment]
👍

**Comment by ingluisjimenez** [comment]
👍

**Comment by mikeklein13** [comment]
👍

**Pull request merged**